opensrp-anc/src/main/java/org/smartregister/anc/library/util/Utils.java

```diff
@@ -853,13 +853,12 @@ public static boolean checkJsonArrayString(String input) {
     /**
      * @param receives iterated keys and values and passes them through translation in nativeform
      *                 to return a string. It checks whether the value is an array, a json object or a normal string separated by ,
-     * @param key
      * @return
      */
     @SuppressLint({"NewApi"})
-    public static String returnTranslatedStringJoinedValue(String value, String key) {
+    public static String returnTranslatedStringJoinedValue(String value) {
         try {
-            if (value.startsWith("[")) {
+            if (value.charAt(0) == '[') {
```

Comment:
This will break if the value is `null` or `empty` Check that `value` is not blank.

Reply:
Resolved